metricbeat/module/docker/vendor/github.com/fsouza/go-dockerclient

```diff
@@ -0,0 +1 @@
+Subproject commit 23c589186c2a92a8742da55f19aec4997dae5cbb
```

Comment:
Looks like the dependencies were added as subproject instead of having it checked in. Run `make clean-vendor` in the directory `github.com/elastic/beats` and it will remove all the versioning files. Afterwards make sure to add the directories and check them in.